addons/cert-manager/template/testgrid/k8s-docker.yaml

```diff
@@ -38,9 +38,12 @@
     sleep 30s
     kubectl get crds
     kubectl get all -n cert-manager
+    kubectl create -f https://raw.githubusercontent.com/cert-manager/website/master/content/docs/tutorials/acme/example/staging-issuer.yaml
+    kubectl get issuers
```

Comment:
i dont think this will ever return an error unless you get a specific issuer

Reply:
right, I'll change it